test_suite/cloud/test_aws.py

```diff
@@ -53,6 +55,102 @@ def test_nouveau_is_blacklisted(self, host):
         assert host.file(file_to_check).contains('blacklist nouveau'), \
             f'nouveau is not blacklisted in "{file_to_check}"'
 
+    def test_unwanted_packages_are_not_present(self, host):
+        unwanted_pkgs = [
```

Comment:
I'm guessing some of this packages are also going to be part of a generic test. Or are all of them specific to aws?

Reply:
AFAIK, they are specific to AWS. However there might be some of them that are common to other clouds.
We won't know until we implement the same tests for other providers.